client/src/templates/Challenges/components/Side-Panel.js

```diff
@@ -35,7 +33,8 @@ const propTypes = {
 };
 
 export class SidePanel extends Component {
-  componentDidMount() {
+  async componentDidMount() {
```

Comment:
```suggestion
  componentDidMount() {
```
I don't think this is necessary, see below.